src/testIndex.ts

```diff
@@ -0,0 +1,11 @@
+const favoriteFruits: string[] = ['apples', 'strawberry', 'orange'];
+function addFruit(fruit: string): void {
+  favoriteFruits.push(fruit);
+  const name = 'Alex';
```

Comment:
I'm sorry, but you can see here - you didn't specify a type for `name` and it's still a string because of `'Alex'`